regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -80,3 +80,9 @@ <h3 class="toc-type">Table of Contents</h3>
 {% endblock %}
 
 {% block chrome-sidebar %}{% endblock %}
+
+{% block preload %}
+<script>
+  window.DOC_NUMBER = '{{ doc_number }}';
+</script>
```

Comment:
It's _effectively_ a global setting when it's in the templates, but only because we only ever _currently_ display one template at a time. This is why global state and singletons are an anti-pattern -- they limit the flexibility of the application in very stringy, hard-to-fix ways, should you ever need to modify their functionality.